drush/cm_config_tools.drush.inc

```diff
@@ -108,6 +108,22 @@ function cm_config_tools_drush_command() {
     'aliases' => array('cmcr'),
   );
 
+  $items['cm-config-tools-suggest'] = array(
+    'description' => "List the dependencies of an extension's default configuration.",
+    'arguments' => array(
+      'module' => 'Module machine name.',
+    ),
+    'options' => array(
+    ),
+    'examples' => array(
+      'drush cm-config-tools-suggest MODULE_NAME' => 'List the dependencies of the configuration',
+    ),
+    'aliases' => array('cmcs'),
+    'outputformat' => array(
```

Comment:
We don't have this bit for the other commands. If we remove it, what does the output look like?